Alignment/OfflineValidation/src/PlotAlignmentValidation.cc

```diff
@@ -541,12 +499,16 @@ void PlotAlignmentValidation::plotSS(const std::string& options, const std::stri
   bool plotLayers = false;  // overrides plotLayerN
   //  bool plotRings  = false;  // Todo: implement this?
   bool plotSplits = false;
+  if (plotSplits)
+    ;
```

Comment:
what's the point of this change?

Reply:
No idea. Removed.

Reply:
Now I remembered. This actually is needed for a reason I have never understood. Compiler seems to throw "[-Werror=unused-but-set-variable" error eventhough 'plotSplits' is assigned in if-statement right after.

Reply:
is `plotSplits` used anywhere else below?

Reply:
yes multiple times in assignments within or out of if-statements and few times in the condition itself

Reply:
ah it is late ... indeed in that member function it is unused. Will remove it entirely in the second commit.

Reply:
Removed.